extensions/functions_geometry.yaml

```diff
@@ -0,0 +1,103 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "st_x"
+    description: >
+      Return the x coordinate of the point.  Return null if not available.
+    impls:
+      - args:
+          - value: geometry
+            name: point
+        return: fp64
+  -
+    name: "st_y"
+    description: >
+      Return the y coordinate of the point.  Return null if not available.
+    impls:
+      - args:
+          - value: geometry
+            name: point
+        return: fp64
+  -
+    name: "st_numpoints"
+    description: >
+      Return the number of points in the geometry.  The geometry should be an st_linestring
+      or st_circularstring.
+    impls:
+      - args:
+          - value: geometry
+            name: geom
+        return: i64
+  -
+    name: "st_isempty"
+    description: >
+      Return true is the geometry is an ampty geometry.
+    impls:
+      - args:
+          - value: geometry
+            name: geom
+        return: boolean
+  -
+    name: "st_isclosed"
+    description: >
+      Return true if the geometry's start and end points are the same.
+    impls:
+      - args:
+          - value: geometry
+            name: geom
+        return: boolean
+  -
+    name: "st_issimple"
+    description: >
+      Return true if the geometry does not self intersect.
+    impls:
+      - args:
+          - value: geometry
+            name: geom
+        return: boolean
+  -
+    name: "st_isring"
```

Comment:
I also agree that the prefix isn't useful here, but I figured i'd just follow the same names as used in PostGIS.  I think @paleolimbot mentioned that it might be better to keep the same names in order to be taken seriously?  I'm open to either though.

Reply:
ahh gotcha...I'll go ahead and drop the prefix in both the open PRs then!